documentation/logs.md

```diff
@@ -172,11 +176,15 @@ The following PowerShell cmdlets include the `-Verbose` argument:
 - `Receive-BoltFile`
 - `Invoke-BoltApply`
 
-Verbose output is useful when you want to see the results for Bolt actions on your targets that are
-usually not printed to standard out (stdout). Verbose isn't a log level, but is a way of telling
-Bolt to output additional information in a human-readable format. Verbose output is particularly
-useful for debugging your tasks and plans - if you're not sure why something is failing, try running
-it with `--verbose` to get more information.
+Verbose output is particularly
+useful for debugging your tasks and plans. If you're not sure why something is failing, try running
+it with `--verbose` to get more information. You can also use `out::verbose` in plans to print messages to stdout in verbose mode. For example, to print "Hello world" to stdout in verbose mode:
```

Comment:
```suggestion
Verbose output is particularly useful for debugging your tasks and plans. If
you're not sure why something is failing, try running it with `--verbose` to get
more information. You can also use `out::verbose` in plans to print messages to
stdout in verbose mode. For example, to print "Hello world" to stdout in verbose
mode:
```

Limit line length in docs to ~80 characters (except in code blocks). There's a VS Code extension called 'Rewrap' that'll do this for you.

Reply:
+1 to Rewrap. Super useful